runtime/testy/studentAPI.py

```diff
@@ -2,20 +2,20 @@
 
 class Robot:
   def __init__(self, toManager, fromManager):
-    self.fromManager = fromManager
-    self.toManager = toManager
+    self.fromManager = fromManager #pipe
```

Comment:
remove the commented out variables.
git allows us to look at past histories if we need to revert.
(make sure to not have whitespace at the end of the line after deleting)